gobblin-salesforce/src/main/java/org/apache/gobblin/salesforce/SalesforceConfigurationKeys.java

```diff
@@ -26,12 +26,12 @@ private SalesforceConfigurationKeys() {
   public static final String BULK_API_USE_QUERY_ALL = "salesforce.bulkApiUseQueryAll";
 
   // pk-chunking
-  public static final String PK_CHUNKING_TEST_BATCH_ID_LIST = "salesforce.pkChunking.testBatchIdList";
-  public static final String PK_CHUNKING_TEST_JOB_ID = "salesforce.pkChunking.testJobId";
+  public static final String BULK_TEST_JOB_ID = "salesforce.bulk.test.job.id";
```

Comment:
Please follow the config naming convention of using '.' as namespace separators, so this should be "salesforce.bulk.testJobId". Change the other new keys to conform.

Reply:
Thanks! Changed to 
```
public static final String BULK_TEST_JOB_ID = "salesforce.bulk.testJobId";
public static final String BULK_TEST_BATCH_ID_LIST = "salesforce.bulk.testBatchIds";
```